Sources/AsyncHTTPClient/ConnectionPool/HTTP1/HTTP1ClientChannelHandler.swift

```diff
@@ -72,11 +72,13 @@ final class HTTP1ClientChannelHandler: ChannelDuplexHandler {
     private let backgroundLogger: Logger
     private var logger: Logger
     private let eventLoop: EventLoop
+    private let eventLoopDescription: Logger.MetadataValue
     private let connectionIdLoggerMetadata: Logger.MetadataValue
 
     var onConnectionIdle: () -> Void = {}
     init(eventLoop: EventLoop, backgroundLogger: Logger, connectionIdLoggerMetadata: Logger.MetadataValue) {
         self.eventLoop = eventLoop
+        self.eventLoopDescription = Logger.MetadataValue(stringLiteral: eventLoop.description)
```

Comment:
This is not a string literal, so we shouldn't use this. Let's use the string interpolation instead.